src/DataProccesing.java

```diff
@@ -0,0 +1,95 @@
+import java.io.*;
+
+public class DataProccesing {
```

Comment:
There should be 2 classes - one for managing cashing machines, other for users. For example - `UserService` and `WachingMachineService`